environments/v2f-prod/terraform/bucket.tf

```diff
@@ -4,8 +4,33 @@ provider google-beta {
   alias = "v2f"
 }
 
+provider vault {
+  alias = "v2f"
+}
+
 resource google_storage_bucket v2f_results_bucket {
   provider = google-beta.v2f
   name = "variant-to-function-result-sets"
   location = "US"
 }
+
+module v2f_writer {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google-beta.v2f,
+    vault.target = vault.v2f
+  }
+
+  account_id = "v2f-writer"
+  display_name = " Service account to write V2F data to GCS"
+  vault_path = "secret/dsde/monster/prod/v2f/service-accounts/bucket-writer"
+  roles = []
+}
+
+resource google_storage_bucket_iam_member v2f_iam {
+  provider = google-beta.v2f
+  bucket = google_storage_bucket.v2f_results_bucket.name
+  role = "roles/storage.objectCreator"
+  member = "serviceAccount:${module.v2f_writer.email}"
+  depends_on = [module.v2f_writer.delay]
+}
```

Comment:
One more piece that will be needed: can you add a block [like this](https://github.com/broadinstitute/monster-deploy/pull/34/files#diff-da4bbae2f2cf9e56b29e2a95166ee2beR31-R36) so Workload Identity will work?